src/components/SigningOfficialDaaAgreementWrapper.jsx

```diff
@@ -1,6 +1,6 @@
 import { Notifications } from '../libs/utils';
 import BroadLibraryCardAgreementLink from '../assets/Library_Card_Agreement_2023_ApplicationVersion.pdf';
-import NIHLibraryCardAgreementLink from '../assets/NIH_Library_Card_Agreement_11_17_22_version.pdf';
+import NihLibraryCardAgreementLink from '../assets/NIHLibraryCardAgreement08012024.pdf';
```

Comment:
I'm told that we don't currently delete old pdfs; I would encourage us to do so. And in general, instead of checking in a file with a different name based on the date, overwrite the previous one. Old files are never lost and the commit history should tell us what version a file is.

Reply:
@pshapiro4broad and I discussed this.  It would probably be good AC for [https://broadworkbench.atlassian.net/browse/DT-1602](https://broadworkbench.atlassian.net/browse/DT-1602)